include/aos/common/cloudprotocol/alerts.hpp

```diff
@@ -0,0 +1,536 @@
+/*
+ * Copyright (C) 2024 Renesas Electronics Corporation.
+ * Copyright (C) 2024 EPAM Systems, Inc.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef AOS_CLOUDPROTOCOL_ALERTS_HPP_
+#define AOS_CLOUDPROTOCOL_ALERTS_HPP_
+
+#include "aos/common/tools/variant.hpp"
+#include "aos/common/types.hpp"
+
+namespace aos {
+namespace cloudprotocol {
+namespace alerts {
```

Comment:
We don't need additional to cloudprotocol namespace. Remove alerts namespace. All names already include alert prefix. For example:

```c++
aos::cloudprotocol::alerts::AlertItem
aos::cloudprotocol::AlertItem
```

And we agreed before to provide namespace per module: cloudprotocol this is module in this case.